Express/model.js

```diff
@@ -20,4 +20,23 @@ const fetchArticleByID = async (article_id) => {
   return rows[0];
 };
 
-module.exports = { fetchAllTopics, fetchArticleByID };
+const fetchArticles = async () => {
+  const { rows } = await db.query(`SELECT articles.article_id,
+                                          articles.author,
+                                          articles.title,
+                                          articles.topic,
+                                          articles.created_at,
+                                          articles.votes,
+                                          articles.article_img_url,
+                                          COUNT(comments.article_id)::INT AS comment_count
+                                   FROM articles /*^casting as I was getting article_id in string format^*/
```

Comment:
Nice use of casting, but there is no need to add a comment explaining your code as it is still very readable here

Reply:
comment is for me, cause I just learned (or recalled if we've went over this during the course) about casting while doing the task